managed/models/database.go

```diff
@@ -949,17 +949,17 @@ var databaseSchema = [][]string{
 			PRIMARY KEY (dump_id, chunk_id)
 		)`,
 	},
-	100: {
+	88: {
 		`DROP TABLE kubernetes_clusters`,
 	},
-	101: {
+	89: {
 		`DROP TABLE IF EXISTS ia_channels`,
 		`DROP TABLE IF EXISTS ia_rules`,
 		`ALTER TABLE ia_templates RENAME TO alert_rule_templates`,
 		`UPDATE settings SET settings = settings #- '{alerting, email_settings}';`,
 		`UPDATE settings SET settings = settings #- '{alerting, slack_settings}';`,
 	},
-	88: {
+	90: {
 		`UPDATE settings SET settings = settings - 'alert_manager_url'`,
 	},
```

Comment:
it was done on purpose to have some room for newer items that might appear in PMM 2.x to not have conflicts

Reply:
Due merging PR's in `v3` branch we got broken order `87, 100, 101, 88`. But we will have conficts with `v2` anyway, because it's same lines, it doesn't matter what number specified there, no?

Reply:
I meant conflicts in runtime. when users in DB will have higher number of schema_migration then we have in the first migration of PMM 3 and it will be skipped. And also each time we create new migration in PMM 2 we will have to update all these values.

Reply:
Ok, changed back to 1xx